File: git-intro/fizzbuzz/julia/fizzbuzz.jl
#=
    To run the code, you must have julia installed
    Go to: https://julialang.org/
    Once installed, open your terminal and type:
        `julia fizzbuzz.jl`
    
    ** Make sure you are in the julia folder so that the
    ** code file can be found
=#

lower = 75
upper = 200

fizz = 14
buzz = 6

for i = lower:upper
    is_fizz = i % fizz == 0
    is_buzz = i % buzz == 0
    if is_fizz && is_buzz
        println("fizzbuzz")
    elseif is_fizz
        println("fizz")
    elseif is_buzz
        println("buzz")
    else
        println(i)
    end
end

File: git-intro/fizzbuzz/java/fizzbuzz.java
/*
 *  To run the code, you must have java installed
    Go to: https://www.oracle.com/java/technologies/downloads
    Once installed, open your terminal and type:
        `java fizzbuzz.java`
    
    Make sure you are in the java folder so that the
     code file can be found
 */

public class fizzbuzz {

    public static void main(String[] args) {
        int lower = 0;
        int upper = 100;

        int fizz = 1;
        int buzz = 2;

        for (int i = lower; i <= upper; i++){
            boolean isFizz = i % fizz == 0;
            boolean isBuzz = i % buzz == 0;

            if(isFizz && isBuzz){
                System.out.println("fizzbuzz");
            } else if(isFizz){
                System.out.println("fizz");
            } else if(isBuzz) {
                System.out.println("buzz");
            } else {
                System.out.println(i);
            }
        }


    }

}